python-in-edu/resources/models.py

```diff
@@ -4,24 +4,52 @@
 from django.core.mail import send_mail
 from django.urls import reverse
 
-from multiselectfield import MultiSelectField
-
 from mysite.settings import DEFAULT_FROM_EMAIL
-from . import choices
 
 
+def get_initial_status():
+    return ResourceStatus.objects.get(sequence=1).id
+
 # Profile Models
+class Organization(models.Model):
+    name = models.CharField(max_length=50)
+    description = models.CharField(max_length=200)
+    creation_timestamp = models.DateTimeField(auto_now_add=True)
+    update_timestamp = models.DateTimeField(auto_now=True)
+    active = models.BooleanField(default=True)
+
+    def __str__(self):
+        return f'{self.name}-{self.description}'
 
 
-class Profile(models.Model):
+class ProfileRole(models.Model):
+    name = models.CharField(max_length=100)
+    active = models.BooleanField(default=True)
+    creation_timestamp = models.DateTimeField(auto_now_add=True)
+    update_timestamp = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return f'{self.name}'
+
+
+class ProfilePopulation(models.Model):
```

Comment:
I think it's better to have a `fixture` file so that we can run `loaddata` to load all the possible values to models `ProfileRole`, `ProfilePopulation` and similar models.
I tried to register, and it raises ` NOT NULL constraint failed: resources_profile.populations_id` and I was not aware that `Profile` table has association with `ProfilePopulation` table, and it is empty.

Reply:
I agree completely with that. I'll work on that as well.

Reply:
Okay. That sounds more handy than having a separate dump process. So, that will be called during db migrations itself?

Reply:
Correct. According to the django docs, the fixtures are really used from a testing perspective whereas initial data loading should be handled via data migrations.

Reply:
That makes perfect sense. Thanks for pointing it out.